projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts

```diff
@@ -5,16 +5,20 @@
 
 import { INotebookTracker } from '@jupyterlab/notebook';
 import { IThemeManager } from '@jupyterlab/apputils';
+import { getVdkCellIndices } from './serverRequests';
 
-const addNumberElement = (number: Number, node: Element) => {
+export const addNumberElement = (number: Number, node: Element) => {
   const numberElement = document.createElement('div');
   numberElement.innerText = String(number);
-  numberElement.classList.add('jp-vdk-cell-num');
-
+  if (node.classList.contains('jp-vdk-failing-cell')) {
+    numberElement.classList.add('jp-vdk-failing-cell-num');
+  } else {
+    numberElement.classList.add('jp-vdk-cell-num');
+  }
```

Comment:
nit: there is a shorter way to deduplicate `numberElement.classList.add(` to once, and have the conditional arg

Reply:
@duyguHsnHsn You can shorten this by using ternary if you're in the mood for code golf. Not sure if it will be more readable, though. Use your best judgement.

Reply:
changed it with ternary